other-teammate

```diff
@@ -1,4 +1,4 @@
-Here is some code a different teammate has worked on. 
+Here is some code a different teammate has worked on. Adding awesome stuff.
 
 Manager knows Engineer wants to get better at XYZ because of botany.
```

Comment:
@botany-demo here is a good case to look into at how to solve X

Reply:
@krichdev testing

Reply:
@botany-lead hello

Reply:
@krichdev you too

Reply:
@krichdev again

Reply:
@krichdev 1 more time

Reply:
@krichdev hello